src/posthog-core.ts

```diff
@@ -130,6 +130,7 @@ export const defaultConfig = (): PostHogConfig => ({
     ip: true,
     opt_out_capturing_by_default: false,
     opt_out_persistence_by_default: false,
+    opt_out_useragent_filter: false,
```

Comment:
A more correct name would be `opt_out_user_agent_filter` with a `_` between `user` and `agent` but kept the spelling consistent with `custom_blocked_useragents` introduced in https://github.com/PostHog/posthog-js/pull/788 /cc @pauldambra

Reply:
The spelling mistake that keeps on giving. Good call 👍